server/src/routes/v1/apps/formatting/convertAppsToApiV1Format.js

```diff
@@ -15,6 +15,9 @@ const convertDbAppViewRowToAppApiV1Object = (app) => ({
 
     name: app.name,
     description: app.description || '',
+
+    hasChangelog: app.has_changelog,
+    hasPlugin: app.has_plugin,
```

Comment:
Should `hasPlugin` also expose the `pluginType` from `d2.config.js`?  That's an immediate win since we have both dashboard and capture plugin types available to day.

Reply:
exposed plugin_type, and now the Plugin tag shows the plugin type if it exists

![image](https://github.com/user-attachments/assets/be987b4d-553d-4cbb-9728-5638f0fe3d5d)